addons/map/README.md

```diff
@@ -8,6 +8,10 @@ Various tweaks to the in-game map, including:
 - Map illumination (optional)
 - Blue Force Tracker (optional)
 
+#### Variables
```

Comment:
Ah, right, line 13 has 2 leading spaces to create a new list level, to serve as a comment for line 12. See the [rich diff](https://github.com/acemod/ACE3/pull/4196/files/f930f58732a8a9f8c612ce61c8efca5decfd3654?short_path=8c8b8b6#diff-8c8b8b6450fc239d39d46113e6fa9862).

I'll remove the note from the README though, it probably shouldn't serve as a dumping ground for _possibly_ interesting hints.